src/renderer/components/ui/sendMoney/SendMoneyForm.js

```diff
@@ -342,12 +342,13 @@ export const SendMoneyForm = ({
         <TextField
           name='memo'
           placeholder={
-            values.recipient.length === 35
+            values.recipient ? values.recipient.length === 35
               ? `You can't include message to transparent address`
```

Comment:
text should be:

Only shielded Zcash addresses (beginning with a 'z') can receive messages